assets/src/scripts/monitoring-location/components/hydrograph/selectors/iv-data.js

```diff
@@ -80,58 +79,70 @@ const transformToCumulative = function(points) {
  */
 export const getIVDataPoints = memoize(dataKind => createSelector(
```

Comment:
we are now only rendering the selected method so this selector only returns the single method.